pubber/src/main/java/daq/pubber/Pubber.java

```diff
@@ -568,12 +566,11 @@ private void processDeviceMetadata(Metadata metadata) {
           configuration.algorithm = authType.value();
         }
       }
+      isGatewayDevice = ifNotNullGet(metadata.gateway.proxy_ids, p -> true, false);
```

Comment:
I think this can be simplified to just "metadata.gateway.proxy_ids != null"?  Or I think there's a "ifNotNullThen" helper function that does the same without the lambda.